test/integration/verify/controls/aws_ssm_resource_compliance_summaries.rb

```diff
@@ -0,0 +1,11 @@
+title 'Test AWS SSM Resource Compliance Summaries in bulk'
+
+control 'aws-ssm-resource-compliance-summaries-1.0' do
+
+  impact 1.0
+  title 'Ensure AWS SSM Resource Compliance Summary has current properties'
+
+  describe aws_ssm_resource_compliance_summaries do
+    it { should_not exist }
```

Comment:
Would this test fail if an SSM compliance summary is created at some point?

Reply:
If this is simply to test the resource instantiation then could use something like e.g. 
```
it { should be_in [true,false] }
```